/imu_fusion:
  ros__parameters:
    gyro_factor: 0.95

/localization_fusion:
  ros__parameters:
    frequency: 200.0
    two_d_mode: true
    imu0: imu/data
    imu0_config: [false, false, false,  # x, y, z
                  true,  true,  true,   # roll, pitch, yaw
                  false, false, false,  # vx, vy, vz
                  true,  true,  true,   # vroll, vpitch, vyaw
                  false, false, false]  # ax, ay, az
    imu0_queue_size: 5
    odom0: odom
    odom0_config: [true, true, false,   # x, y, z
                  false, false, false,  # roll, pitch, yaw
                  false, false, false,  # vx, vy, vz
                  false, false, true,   # vroll, vpitch, vyaw
                  false, false, false]  # ax, ay, az
    odom0_queue_size: 5

/rollover_detect:
  ros__parameters:
    roll_threshold: 60.0
    pitch_threshold: 60.0
    event_interval: 1.0

/collision_alert:
  ros__parameters:
    distance_threshold: 0.1
    event_interval: 1.0
